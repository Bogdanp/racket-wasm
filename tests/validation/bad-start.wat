(module
 (func
  (result i32)
  (i32.const 42))
 (start 0))