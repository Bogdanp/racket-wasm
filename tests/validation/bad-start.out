(#f . "start function must have type [] -> []\n  found: [] -> [i32]\n  at: \"start function\"")